src/client/components/RegistrationConsents.tsx

```diff
@@ -25,11 +27,28 @@ export const RegistrationConsents = ({
 	geolocation,
 	useIdapi,
 	noMarginBottom,
+	appName,
 }: RegistrationConsentsProps) => {
-	// don't show the Saturday Edition newsletter option for US and AUS
-	const showSaturdayEdition = !(['US', 'AU'] as GeoLocation[]).some(
-		(location: GeoLocation) => location === geolocation,
-	);
+	// don't show the Saturday Edition newsletter option for US and AUS or if using Feast app
+	const showSaturdayEdition = (() => {
+		const isValidLocation = !(['US', 'AU'] satisfies GeoLocation[]).some(
+			(location: GeoLocation) => location === geolocation,
+		);
+
+		const isValidAppName = !(['Feast'] satisfies AppName[]).some(
+			(name: AppName) => name === appName,
+		);
+
+		return isValidLocation && isValidAppName;
+	})();
+
+	// show the Feast newsletter option if the app is Feast and Saturday Edition is not shown
+	const showFeast =
```

Comment:
This is quite convoluted when it could just be `appName === 'Feast' && !showSaturdayEdition` but I suppose we could use it for other appname checks?

Reply:
Updated this to make it just related to the Feast app. If we need more apps in the future we can always change it.